build-tools/scripts/BuildEverything.mk

```diff
@@ -92,9 +92,18 @@ $(TASK_ASSEMBLIES): bin/%/lib/xbuild/Xamarin/Android/Xamarin.Android.Build.Tasks
 	$(MSBUILD) $(MSBUILD_FLAGS) /p:Configuration=$* $(_MSBUILD_ARGS) $(SOLUTION)
 
 $(FRAMEWORK_ASSEMBLIES):
-	$(foreach a, $(API_LEVELS), \
+	PREV_VERSION="v1.0"; \
+	for a in $(API_LEVELS); do \
+		CUR_VERSION=`echo "$(ALL_FRAMEWORKS)"|tr -s " "|cut -d " " -s -f $${a}`; \
```

Comment:
it wouldn't, because make variables are resolved before executing the shell commands and thus $$a would be empty in this case